mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/CustomClientSessionFlushScenario.kt

```diff
@@ -14,14 +14,16 @@ internal class CustomClientSessionFlushScenario(config: Configuration,
                                                 context: Context) : Scenario(config, context) {
 
     override fun run() {
```

Comment:
This scenario now only creates one session, as it is supposed to test that a manual session is cached on disk then delivered.